src/main/java/org/folio/innreach/domain/service/impl/InnReachRecallUserServiceImpl.java

```diff
@@ -60,4 +61,10 @@ private CentralServer fetchCentralServerWithRecallUser(UUID centralServerId) {
       .orElseThrow(() -> new EntityNotFoundException("Central server with id " + centralServerId + " not found"));
   }
 
+  @Override
+  public InnReachRecallUser getRecallUserForCentralServer(String centralCode) {
+    return centralServerRepository.fetchOneByCentralCode(centralCode)
+      .map(CentralServer::getInnReachRecallUser)
+      .orElseThrow(() -> new EntityNotFoundException("Recall user is not set for central server with code = " + centralCode));
+  }
```

Comment:
apply this method to the service where it was copied and remove the duplicate
https://github.com/folio-org/mod-inn-reach/blob/2cf1c2fcda1733fd304aafe01b29733c8f2a1c7a/src/main/java/org/folio/innreach/domain/service/impl/CirculationServiceImpl.java#L489-L493

Reply:
I can't use this method because it is private

Reply:
Do you mind if i create this method like public ?